include/swift/AST/DiagnosticsSIL.def

```diff
@@ -864,14 +864,16 @@ NOTE(sil_referencebinding_inout_binding_here, none,
 
 // Warnings arising from the flow-sensitive checking of Sendability of
 // non-Sendable values
-WARNING(consumed_value_used, none,
-        "non-Sendable value consumed, then used at this site; could yield race with another thread", ())
 WARNING(arg_region_consumed, none,
         "this application could pass `self` or a Non-Sendable argument of this function to another thread, potentially yielding a race with the caller", ())
 WARNING(consumption_yields_race, none,
-        "non-Sendable value sent across isolation domains here, but could be accessed later in this function (%0 access site%select{|s}1 displayed%select{|, %3 more hidden}2)", (unsigned, bool, bool, unsigned))
+        "non-sendable value sent across isolation domains here, but could be accessed later in this function (%0 access site%select{|s}1 displayed%select{|, %3 more hidden}2)",
```

Comment:
maybe drop the "here" and "this"?
```suggestion
        "non-sendable value sent across isolation domains that could be accessed concurrently in function (%0 access site%select{|s}1 displayed%select{|, %3 more hidden}2)",
```

Maybe wording it like this makes the reader aware that it's an actual potential concurrency issue?
The "this" and "that" don't add much so I suggested removing

Reply:
I sort of like "later" since it implies to me that it's some operation that can happen after the one marked. I also think we should make it present tense so it sounds less hypothetical, i.e., "non-sendable value sent across isolation domains that can be accessed later in function"

Reply:
`non-sendable value sent across isolation domains that could be accessed concurrently later in function`?